File: Year 2/Semester 4/MPP/SeventhWeekServerClient/Persistance/src/main/java/SeventhWeekServerClient/persistance/repository/ITicketRepository.java
package SeventhWeekServerClient.persistance.repository;

import SeventhWeekServerClient.domain.Entity;
import SeventhWeekServerClient.domain.Ticket;

public interface ITicketRepository <ID, E extends Entity<ID>> extends IRepository<ID, E> {
    void addTicket(Ticket ticket);
}

File: Year 2/Semester 4/Programming and Projection Environments/Week 8/SeventhWeekCSharpServerClient/persistence/interfaces/ITicketRepository.cs
using SeventhWeekCSharpServerClient.domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SeventhWeekCSharpServerClient.repository.interfaces
{
    public interface ITicketRepository : IRepository<int, Ticket<int>>
    {
        void AddTicket(Ticket<int> ticket);
    }
}
